src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,121 @@
 package core.basesyntax;
 
+import java.util.Map;
+import java.util.Objects;
+
+@SuppressWarnings("unchecked")
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static int DEFAULT_CAPACITY;
+    private static float LOAD_FACTOR;
+    private int threshold;
+    private Entry<K, V>[] table;
+    private int size;
+
+    public MyHashMap() {
+        DEFAULT_CAPACITY = 16;
+        LOAD_FACTOR = 0.75f;
```

Comment:
you don't need to initialize these two here, return the previous version of these two